pkg/agent/cluster.go

```diff
@@ -70,7 +70,12 @@ func NewCluster(ctx context.Context, assetDir, rendezvousIP, kubeconfigPath, ssh
 	czero := &Cluster{}
 	capi := &clientSet{}
 
-	restclient, err := NewNodeZeroRestClient(ctx, rendezvousIP, sshKey)
+	authToken, err := FindAuthTokenFromAssetStore(assetDir)
+	if err != nil {
+		logrus.Fatal(err)
```

Comment:
What's the reason to log a failure but continue the execution?

Reply:
Nice catch! We should fail if the asset store is not found. Maybe we should also stop the execution here? https://github.com/openshift/installer/blob/fcaa07d1553c1b2cfed84642e7b38693c4af7ec9/cmd/openshift-install/agent/waitfor.go#L69-L72